docs/plat/rpi3.rst

```diff
@@ -237,6 +237,13 @@ The following build options are supported:
   in AArch64 mode. If set to 1, it will jump to BL33 in Hypervisor in AArch32
   mode.
 
+- ``BL32``: This port can load and run OP-TEE image.
```

Comment:
Suggestion: please also add information about where OP-TEE is loaded, for example to memory layout table

```
0x00000000 +-----------------+
           |       ROM       | BL1
0x00010000 +-----------------+
           |       FIP       |
0x00200000 +-----------------+
           |                 |
           |       ...       |
           |                 |
0x01000000 +-----------------+
           |     Kernel      |
           +-----------------+
           |                 |
           |       ...       |
           |                 |
0x02000000 +-----------------+
           |       DTB       |
           +-----------------+
           |                 |
           |       ...       |
           |                 |
0x10000000 +-----------------+
           |   Secure SRAM   | BL2, BL31
0x10100000 +-----------------+
           |   Secure DRAM   | BL32 (Secure payload)
0x10300000 +-----------------+
           | Non-secure DRAM | BL33
0x11000000 +-----------------+
           |                 |
           |       ...       |
           |                 |
0x3F000000 +-----------------+
           |       I/O       |
0x40000000 +-----------------+
```